recipes/GFDL_SPEAR_SICDA/meta.yaml

```diff
@@ -0,0 +1,22 @@
+title: "GFDL SIC DA"
+description: "model ensemble mean state vectors and diagnostics, and input observations from SPEAR sea ice concentration (SIC) data assimilation (DA) run"
+pangeo_forge_version: "0.8.3"
+pangeo_notebook_version: "2021.12.02"
+recipes:
+  - id: sic-da
+    object: "recipe:recipe"
```

Comment:
@William-gregory, which one of the recipes in your `recipe.py` corresponds to `sic-da`?

As described in

> https://pangeo-forge.readthedocs.io/en/latest/introduction_tutorial/intro_tutorial_part3.html#recipes-section

the `object` field here tells Pangeo Forge Cloud which recipe to import for `sic-da`.

For example, if `sic-da` corresponds to your recipe named `recipe1`, then you would specify

```yaml
object: "recipe:recipe1"
```
which means "`sic-da` corresponds to the variable named `recipe1` in the module named `recipe.py`.

Reply:
As it stands, I don't believe there is a variable named simply `recipe` (without a trailing integer) in your `recipe.py`, so nothing will be imported for the test of `sic-da`.

Reply:
Ah ok I I think I understand. Should it be something like:

```
recipes:
  - id: sic-obs
    object: "recipe:recipe1"
  - id: sic-da-forecasts
    object: "recipe:recipe2"
  - id: sic-da-increments
    object: "recipe:recipe3"
  - id: sic-da-priors
    object: "recipe:recipe4"
  - id: sic-da-posteriors
    object: "recipe:recipe5"
```

All within the same `meta.yaml` file?

Reply:
exactly!

Reply:
Got it! I'll update it now